### PR TITLE
[Footer] Componentize the footer

### LINKED ISSUE #44: [Footer] Componentize the footer
Currently we use lots of controllers + templates but we should defer to use components where we can.

We should componentize the footer to be a component rather have it directly in the template.

Example from the Octane repo: https://github.com/patocallaghan/realworld-starter-kit/blob/master/app/templates/components/footer.hbs

### Footer
image

### PR DESCRIPTION
**Pull request by delusioninabox**
Fixes #44 

- [x] Moves footer content into component
- [x] Link to home clears query params
- [x] Component integration test

### CODE REVIEW

File: app/templates/components/footer-bar.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  {{#link-to "home"  (query-params author=null tag=null page=1) class="logo-font" }}
```
Comment (Alonski) [inline app/templates/components/footer-bar.hbs]: Hope this isn't too annoying but there are two spaces between `"home"` and `(query-params...` 😅

**Comment by patocallaghan** [comment]
Awesome work @delusioninabox 👏 Looks great 🎉 
 
I'd add to @Alonski's comment about adding acceptance tests, I'd also probably stop at checking if the footer appears on the page once. Personally I don't think there's much value in adding a test for every page. Also we probably don't need the component integration test as it's not really testing much. Happy to hear if folks disagree though.

**Comment by mansona** [comment]
@patocallaghan I wonder if it's worth getting this merged and write an issue for adding the footer acceptance test? 🤔 I'm just thinking about momentum here

**Comment by patocallaghan** [comment]
@mansona sure 🚢

**Comment by Alonski** [comment]
Good idea @mansona

**Pull request merged**